### ISSUE TITLE
Bot should PM players any unacknowledged warnings when trying to join

### ISSUE THREAD

**Issue by nyuszika7h** [lykoss/lykos#236]
Suggested by @michagogo on IRC.

**Comment by Vgr255** [comment]
IMO, making the players having to do `!warn list` helps ensure that they really know their warning (and don't just blindly do `!ack #` where # is just whatever they see in PM. But maybe that's not necessary, I don't know. I'm +0 anyway.

**Comment by Michagogo** [comment]
Well, on one hand that may be true, but I feel like sending the full
warning text in the message won't really hurt the issue. In fact, it may
even improve it. I feel like the player may be more likely to actually read
the message if it's sent to him in order to understand what's going on.
With the current one, I have a feeling people will read the message, see
what's happening, and at that point (before seeing the warning itself) go
into clickthrough/"clear this"/"get out of the way" mode and be more likely
to list just to find the number.

**Comment by Fudster** [comment]
I agree with vgr. But if the repo maintainers want this I can work on it.
